docs/spark.rst

```diff
@@ -30,7 +30,10 @@ When installing Horovod for usage with Spark, use the extra ``[spark]`` to insta
 
 .. code-block:: bash
 
-    $ ... pip install horovod[spark]
+    pip install "horovod[spark]"
+
+Note that the other Horovod dependencies (e.g., `CMake <https://cmake.org/install/>`__) should also be installed
+as described in `Install <https://github.com/horovod/horovod#install>`__.
```

Comment:
It's a good idea to point to the general Horovod installation guide, but I'd suggest not to link to GitHub here, if possible, but to point to `install_include.rst` in this docs folder. Then the documentation builds served on readthedocs.com would be self consistent.